app/views/bookings/index.html.erb

```diff
@@ -4,5 +4,6 @@
   <p><%= booking.id %></p>
   <p><%= animal.name %></p>
   <p><%= user.email %></p>
+  <%= link_to 'delete', booking_path(booking), method: :delete, data: { confirm: 'Are you sure?' } %>
```

Comment:
also add link to booking#show page so that users can see individual bookings?

Reply:
Added this link, and updated the display fields to provide a bit more information and format them better.